### PR TITLE
Statistics Scroll To Results Added

### PR DESCRIPTION
**Pull request by danny98m**
After the plot is created and the statistics generated the screen will scroll to the displayed results.

### CODE REVIEW

File: lexos/static/js/scripts_statistics.js [hunk 1]
```diff
@@ -207,9 +216,13 @@ $(function () {
         generateStatsBoxPlot()
         // Display the result.
         $('#corpus-stats-result').css({'display': 'block'})
+        $('html, body').animate({
+          scrollTop: $('#get-stats').offset().top
+        }, 1000)
       } else { // Else hide the corpus stats result div.
-        $('#corpus-stats-result').css({'display': 'none'})
+        $('#file-stats-result').css({'display': 'none'})
```
Comment (Weiqi97) [inline lexos/static/js/scripts_statistics.js]: Why does this line needs to get changed?

File: lexos/static/js/scripts_statistics.js [hunk 2]
```diff
@@ -207,9 +216,13 @@ $(function () {
         generateStatsBoxPlot()
         // Display the result.
         $('#corpus-stats-result').css({'display': 'block'})
+        $('html, body').animate({
```
Comment (Weiqi97) [inline lexos/static/js/scripts_statistics.js]: Maybe Use your function call here?

File: lexos/static/js/scripts_statistics.js (hunk 1 above)
Comment (danny98m) [reply lexos/static/js/scripts_statistics.js]: MY BAD. This absolutely does not need to be changed. That was an accident left over from testing something.

**Comment by Weiqi97** [comment]
This PR is merged in #786

**Pull request closed without merging**